src/apispec/ext/marshmallow/openapi.py

```diff
@@ -91,8 +91,11 @@
     "required",
     "enum",
     "type",
-    "items",
     "allOf",
+    "oneOf",
+    "anyOf",
+    "not",
```

Comment:
This is not implemented yet, @stefanv would you prefer working on it in this PR, or remove?

```suggestion
```